### DISCUSSION TITLE
[Microservices] Let the user choose the release strategy

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#226] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi guys,
it would be useful to let the user choose the release strategy to be used for the specific microservice.
Usually, the preferred one is the default one too, `RollingUpdate`, but sometimes we need to use the `Recreate` strategy and at the moment it's not possible (am I wrong?) without using either:
* advanced configurations;
* kustomize (that must be configured on the project.

What do you think about that?

Thanks

**Comment by davidebianchi** [comment]
Hi @silversoul93! As you said, it is possible with kustomize in the actual flow.
It could be added, maybe it is needed to add some advanced configuration on services to add this types of highly specific configuration.

**Reply by frengooo** [reply]
Ok, I'll create a task for that. Thanks 😁

**Reply by frengooo** [reply]
p.s. we easily fixed using Kustomize
